ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -39,6 +40,11 @@
 
   private static final int ARRAY_CAPACITY_INCREMENT = 12;
   private static final int DEFAULT_BACKGROUND_COLOR = Color.TRANSPARENT;
+  private static final String PROP_BACKFACE_VISIBILITY_HIDDEN = "hidden";  
+  private static final String PROP_DECOMPOSED_MATRIX_ROTATE_X = "rotateX";
+  private static final String PROP_DECOMPOSED_MATRIX_ROTATE_Y = "rotateY";
+  private static final float FRONTFACE_VISIBILITY_MIN_ROTATION = -90.f;
+  private static final float FRONTFACE_VISIBILITY_MAX_ROTATION = 90.f;
```

Comment:
Minor nit: these don't really need to be constants. It makes the code less readable and we're not likely to change their value any time soon, and their use is pretty limited.